LernDeutsch-Backend/Program.cs

```diff
@@ -1,18 +1,85 @@
 using LernDeutsch_Backend.Data;
 using LernDeutsch_Backend.Models.Identity;
+using LernDeutsch_Backend.Repositories.Identity;
+using LernDeutsch_Backend.Services.Identity;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.OpenApi.Models;
+using System.Security.Cryptography;
+using System.Text;
 
 var builder = WebApplication.CreateBuilder(args);
 var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+ConfigurationManager configuration = builder.Configuration;
+configuration["JWT:Secret"] = Convert.ToBase64String(RandomNumberGenerator.GetBytes(64));
 // Add services to the container.
 
 builder.Services.AddDbContext<ApplicationDatabaseContext>(x => x.UseSqlServer(connectionString));
-builder.Services.AddIdentityCore<BaseUser>().AddEntityFrameworkStores<ApplicationDatabaseContext>();
+
+builder.Services.AddIdentity<BaseUser, IdentityRole>()
+    .AddEntityFrameworkStores<ApplicationDatabaseContext>()
+    .AddDefaultTokenProviders();
+
+builder.Services.AddAuthentication(options =>
```

Comment:
See commit: 2dbbb4e7b28afc4ad0bd16f6759784daf7f3a380

Reply:
Small mistake which was made , pushed the changes to the [FEATURE] [LER-56] Create missing base services #7 instead of [FEATURE] [LER-56] Create missing base services #5